dissect/target/tools/utils.py

```diff
@@ -261,3 +263,35 @@ def wrapper(*args, **kwargs):
             raise
 
     return wrapper
+
+
+def args2uri(targets: list[str], loader_name: str, rest: list[str]) -> list[str]:
+    """Converts argument-style -L to URI-style
+
+    Turns:
```

Comment:
Does this show properly in Sphinx?

Reply:
![Uploading image.png…]()

Reply:
I keep being confused by what the proper Sphinx/rst syntax is... Sometimes you need `::` and sometimes a single `:` but an indentation also works... Anyway thanks for verifying!